core/cap-0030.md

```diff
@@ -24,19 +24,11 @@ This proposal will also make the CAP-0023 (Two-Part Payments) implementation sim
 The Stellar Network should facilitate simplicity and interoperability with other protocols and networks.
 
 ## Specification
-
-1. Remove every `*_NO_ISSUER` result from `Stellar-transaction.x` 
-2. For every operation that can return a `*_NO_ISSUER` result, remove the issuer check code along with the result.
-3. For the `ChangeTrustOp`, remove all `CHANGE_TRUST_NO_ISSUER` results. When checking the `AUTH_REQUIRED` flag on the issuer, check if the issuer exists first. If not, the `TrustLine` will not be authorized on creation.
-```c++
-if (issuer && !isAuthRequired(issuer))
-{
-    tl.flags = AUTHORIZED_FLAG;
-}
-```
+1. For every operation that can return a `*_NO_ISSUER` result except `ChangeTrustOp`, don't load the issuer or return the `*_NO_ISSUER` result.
+2. Don't include the issuer accounts in the accounts to prefetch for operations that don't depend on the issuer.
```

Comment:
This is purely an implementation detail, no need to include it here.

Reply:
@jonjove removed